sklearn/datasets/base.py

```diff
@@ -672,7 +693,9 @@ def load_boston(return_X_y=False):
     data : Bunch
         Dictionary-like object, the interesting attributes are:
         'data', the data to learn, 'target', the regression targets,
-        and 'DESCR', the full description of the dataset.
+        'DESCR', the full description of the dataset,
+        and 'filename' (added in version 0.20), the physical location
+        of boston csv dataset.
```

Comment:
I am not in favor of  `(added in version 0.20)` in the narrative.

I would add a directive on the bottom of data

```python
        the physical location of iris csv dataset.

        .. versionadded:: 0.20
           The ``filename`` attribute.
```